cmd/retrieve.go

```diff
@@ -96,6 +113,80 @@ func Retrieve(cmd *cobra.Command, _ []string) (err error) {
 		filter.PageNames = pageNames
 	}
 
+	var sinceStr string
+	since := time.Now()
+	hasSince := false
+	if sinceStr, err = cmd.Flags().GetString(flags.Since.Name); err != nil {
+		return err
+	} else if len(sinceStr) > 0 {
+		// First try to parse something like "01/02 03:04:05PM '06 -0700"
```

Comment:
I know that most of the users are from the US timezone but I would like to know how are we handling the different timezones. I retrieved by setting since with the datetime of my computer and I retrieved the entire site because it has an offset of 5 hours (18:21 -> 23:21)
![image](https://user-images.githubusercontent.com/30167939/228090959-040af8e4-f37f-4892-88f9-89d58520ee3f.png)

Reply:
Interesting. It didn't do that for me, but I may need to make sure we convert timestamps from the local time to UTC before doing the comparison in SQL.